build/dependencies.yaml

```diff
@@ -64,7 +64,7 @@ dependencies:
 
   # etcd
   - name: "etcd"
-    version: 3.5.12
+    version: 3.5.13
```

Comment:
```suggestion
    version: 3.5.12
```

Reply:
To fix https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/124026/pull-kubernetes-dependencies/1775463778079477760

Reply:
ping @liangyuanpeng could you resolve the comment?

Reply:
@ahrtr @serathius  I will work  for it in this weekend,Thanks.